Dockerfile

```diff
@@ -1,2 +1,5 @@
 FROM yastdevel/cpp
 COPY . /usr/src/app
+
+# Install tmux to make sure the libyui+YaST integration tests are run
+RUN zypper --non-interactive in tmux
```

Comment:
I think this could be moved between FROM and COPY because tmux will change less often than our git and docker caching will slightly improve.